### PR TITLE
prince hypermart added

### PR DESCRIPTION
**Pull request by zahra-abdullah**
https://github.com/osmlab/name-suggestion-index/issues/4125

### CODE REVIEW

File: brands/shop/houseware.json [hunk 1]
```diff
@@ -175,6 +175,18 @@
         "shop": "houseware"
       }
     },
+    {
+      "displayName": "Prince Hypermart",
+      "id": "princehypermart-52d9d6",
+      "locationSet": {"include": ["ph", "phl"]},
```
Comment (bhousel) [inline brands/shop/houseware.json:181]: Just `ph` is ok here.. (`phl` is the same thing, but the 3 letter code)
```suggestion
      "locationSet": {"include": ["ph"]},
```

**Review by bhousel — CHANGES_REQUESTED** [review]
just one minor change, then we're good to merge it...

File: brands/shop/houseware.json (hunk 1 above)
Comment (zahra-abdullah) [reply brands/shop/houseware.json]: updated! please review thanks!

**Comment by bhousel** [comment]
Looks Good @zahra-abdullah , thanks for the quick fix, and welcome 👋

**Pull request merged**